docs/source/guide/api_migrate.md

```diff
@@ -0,0 +1,181 @@
+- `aggregated` is deprecated
+- `aggregated_completed_by` is deprecated
+- `aggregated_ids` is deprecated
+- `ground_truth` is deprecated
+- `result` is no longer a double list `"result": [[... ]]` and is now a single list `“result”: [...] `
+
+#### Previous version response
+
+```json
+"completions": [ 
+ {
+ “----DEPRECATED SECTION---
+   "aggregated": true,
+   "aggregated_completed_by": [
+     103
+   ],
+   "aggregated_ids": [
+     800955
+   ],
+   "ground_truth": false,
+---- END OF DEPRECATED SECTION
+   "result": [
+     [ 
+       {
+         "id": "7tHQ-n6xfo",
+         "type": "choices",
+         "value": {
+           "choices": [
+             "Neutral"
+           ]
+         },
+         "to_name": "text",
+         "from_name": "sentiment"
+       }
+     ]
+   ]
+ }
+]
+```
+
+#### Current version response
+
+
+```json
+"annotations": [ 
+ {
+   "result": [
+       {
+         "id": "7tHQ-n6xfo",
+         "type": "choices",
+         "value": {
+           "choices": [
+             "Neutral"
+           ]
+         },
+         "to_name": "text",
+         "from_name": "sentiment"
+       }
+   ]
+ }
+]
+```
+
+## Invite people and create accounts
+
+When you invite people to join your organization, workspace (formerly team), and projects, there are many changes. 
+
+### Update tokens in use
+The old tokens are no longer supported. Make a request reset your organization token used to create invitation links. 
+
+See the [reset organization token API endpoint documentation](/api#operation/api_invite_reset-token_create).
+
+### Updated invite to organization endpoint
+
+The endpoint that you use to generate invitations to your Label Studio Enterprise organization has changed from `https://app.heartex.ai/organization/welcome/<token>` to `http://localhost:8000/api/invite`.
+
+See the [organization invite link API endpoint documentation](/api#operation/api_invite_list). 
+
+### Updated flow for inviting project members
+
+Links that invite people directly to projects are no longer supported. Instead, perform the following steps in order:
+1. Invite the person to the organization with the [link returned by the organization invite API endpoint](/api#operation/api_invite_list).
+2. Change the user's role to annotator with the [organization membership role endpoint](/api#operation/api_organizations_memberships_partial_update).
+3. Add the user to the project by making a [POST request to the project member endpoint](/api#operation/api_projects_members_create).
+
+
+## Create and update external and cloud storage
+Some endpoints have been updated and some payload parameters are different when performing actions with the storage API.
+
+### Updates to storage endpoints
+
+When you want to retrieve information about a storage configuration, specify the type of storage in the API endpoint. 
+
+Instead of `/api/storages/<int:pk>/`, call `api/storages/s3/` for Amazon S3 storage connections across the organization.
```

Comment:
int:pk -> storage_id 
api/storages/s3/ -> api/storages/s3/<storage_id>

Reply:
this is to get a specific storage ID, but you can also call it without storage ID to get all storages for the organization